src/frontend/overlay/src/components/organisms/widgets/Queue.js

```diff
@@ -131,9 +131,9 @@ export const Queue = ({ widgetData }) => {
             setFeaturedRunLoaded(false);
         }
     }, [hasFeatured]);
-    for (let queueEntry of _.sortBy(queue, ["isFirstSolvedRun"])) {
+    for (let queueEntry of _.sortBy(queue, function (e) { return (e.result === undefined) ? false : e.result.isFirstToSolveRun; })) {
```

Comment:
[(e) => e.result?.isFirstToSolveRun ?? false]
Second argument should be an array
Although im not sure returning false leaves them in place. (won't it be so that in progress runs will be at the bottom or at the top of the queue?)